### PR TITLE
Prettier to recipes folder

### PR DESCRIPTION
**Pull request by CodinCat**
#2385

### CODE REVIEW

File: docs/recipes/UsingImmutableJS.md [hunk 1]
```diff
@@ -168,26 +168,26 @@ Never let a plain JavaScript object contain Immutable.JS properties. Equally, ne
...
 
-```
+```js
 // avoid
-const newObj = { key: value};
-const newState = state.setIn(['prop1’], newObj); // <-- newObj has been added as a plain
-    // JavaScript object - NOT as an Immutable.JS Map
+const newObj = { key: value }
+const newState = state.setIn(['prop1'], newObj) // <-- newObj has been added as a plain
+// JavaScript object - NOT as an Immutable.JS Map
```
Comment (CodinCat) [inline docs/recipes/UsingImmutableJS.md]: This comment may need some fixing. Also at line 190

File: docs/recipes/ReducingBoilerplate.md [hunk 2]
```diff
@@ -347,12 +348,7 @@ The middleware that interprets such actions could look like this:
 ```js
 function callAPIMiddleware({ dispatch, getState }) {
   return next => action => {
-    const {
-      types,
-      callAPI,
-      shouldCallAPI = () => true,
-      payload = {}
-    } = action
+    const { types, callAPI, shouldCallAPI = () => true, payload = {} } = action
```
Comment (timdorr) [inline docs/recipes/ReducingBoilerplate.md]: Another one where we should leave it alone.

File: docs/recipes/ComputingDerivedData.md [hunk 3]
```diff
@@ -108,24 +104,21 @@ import { toggleTodo } from '../actions'
...
       dispatch(toggleTodo(id))
     }
   }
 }
 
-const VisibleTodoList = connect(
-  mapStateToProps,
-  mapDispatchToProps
-)(TodoList)
+const VisibleTodoList = connect(mapStateToProps, mapDispatchToProps)(TodoList)
```
Comment (timdorr) [inline docs/recipes/ComputingDerivedData.md]: More connect calls that should be left alone.

File: docs/recipes/UsingImmutableJS.md (hunk 1 above)
Comment (timdorr) [inline docs/recipes/UsingImmutableJS.md]: Yeah, that's weird formatting to begin with. I'd do this:
```js
// avoid
const newObj = { key: value }
const newState = state.setIn(['prop1'], newObj) 
// newObj has been added as a plain JavaScript object, NOT as an Immutable.JS Map

// recommended
const newObj = { key: value }
const newState = state.setIn(['prop1'], fromJS(newObj)) 
// newObj is now an Immutable.JS Map
```

File: docs/recipes/WritingTests.md [hunk 4]
```diff
@@ -371,7 +368,9 @@ In order to be able to test the App component itself without having to deal with
 import { connect } from 'react-redux'
 
 // Use named export for unconnected component (for tests)
-export class App extends Component { /* ... */ }
+export class App extends Component {
+  /* ... */
+}
```
Comment (timdorr) [inline docs/recipes/WritingTests.md]: I think this one you can revert too.

Comment (timdorr) [inline docs/recipes/WritingTests.md]: And the one above.

**Comment by CodinCat** [comment]
Some weird GitHub bug always shows the comment on ComputingDerivedData.md Line 121 which I've fixed..

**Pull request merged**